vocode/streaming/streaming_conversation.py

```diff
@@ -183,9 +195,10 @@ async def process(self, transcription: Transcription):
                 )
                 self.output_queue.put_nowait(event)
                 self.conversation.mark_last_final_transcript_from_human()
-            # else:
-            #     self.kill_tasks_when_human_is_talking()
-            #     self.conversation.broadcast_interrupt()
+                self.conversation.is_human_speaking = False
+            else:
```

Comment:
we should comment this back out - it was a untested experiment on my side

Reply:
should be good now